rose/client/car.py

```diff
@@ -32,7 +33,7 @@ def draw(self, surface):
 
     def draw_label(self, surface, (x, y)):
         font = pygame.font.SysFont(pygame.font.get_default_font(), 20)
-        text = font.render(self.name, 1, (0,0,0))
+        text = font.render(self.name, 1, (0, 0, 0))
```

Comment:
Changes are good, but this code is being removed in #186 - lets wait until this patch is merged before investing more time in the client package.